modules/stanford_basic_page_types/templates/components/basic_page_types_vertical_teaser/basic-page-types-vertical-teaser.twig

```diff
@@ -0,0 +1,71 @@
+{#
```

Comment:
What does this do that a regular card doesn't do? is there a way to use another pattern? we have a couple of card patterns. the regular card is what we use for publications.

Reply:
Most notably, the design calls for the basic page type to be listed between the image at the top of the card and the page title.  Because of that, it needs an extra region (particularly if we're going to use the basic page types breadcrumb-style, which is what Ishi indicated to me was her future plan). I could maybe get it to work with the Super Headline region, if I override the styles.  Again, I was going off the example of the News vertical teaser pattern

Reply:
That's exactly how the publications is using the normal card template. So it can be done in the same way if that's the primary thing.
To me the publications cards and the basic page cards should look identical for that field. It's basically the same type of information for the user.

Reply:
The other thing that this does is that it provides logic for using either the `page_image` which is our metadata field, -OR- the page banner, if one exists and there is no page_image metadata.  That was to ensure backwards compatibility with the way the Card display currently works prior to the addition of the new fields.

In order to preserve that compatibility, I need to use a custom twig file, right? Or should I just do some kind of preprocessing hook?

Reply:
Fixed to use the standard card template.